blueprints/data-solutions/dataproc-lifecycle-via-composer/terraform/variables.tf

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "project_id" {
+  description = "Project ID where all the resources will be created, including; Cloud Composer Environment, dataproc cluster, policies, buckets, etc."
+  default     = "pso-amex-data-platform"
+  type        = string
+}
+
+variable "region" {
+  description = "Region where all the resources will be created, including; Cloud Composer Environment, dataproc cluster, policies, buckets, etc."
+  default     = "us-central1"
+  type        = string
+}
+
+variable "zone" {
+  description = "Zone where all the resources will be created, including; Cloud Composer Environment, dataproc cluster, policies, buckets, etc."
+  default     = "us-central1-a"
+  type        = string
+}
+
+variable "network" {
```

Comment:
can you bundle all network-related variables into a single `vpc_config` variable? does this example also support Shared VPC? (it should)

Reply:
Shared VPC support added, vpc_config variable added.